### DISCUSSION TITLE
error to download, HELP!

### DISCUSSION THREAD

**Discussion by SantyHerrera** [pear-devs/pear-desktop#2284] [Q&A] [answered]
![Captura de pantalla 2024-07-28 103625](https://github.com/user-attachments/assets/13729e2e-a41c-41f9-bebc-402b6d69ef7b)

**Comment by Wpnnt** [comment]
I have the same error

**Comment by JellyBrick** [comment, marked as answer]
Fixed at v3.5.1

**Reply by Wpnnt** [reply]
Thanks!
